src/css/App.css

```
@@ -0,0 +1,145 @@
+.App {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	height: 100vh;
+}
+
+.container {
+	padding-right: 20px;
+	padding-left: 20px;
```

Для такого используем короткий формат

```suggestion
	padding: 0 20px;
```